bom/build.gradle

```diff
@@ -0,0 +1,39 @@
+description = 'gRPC: BOM'
+
+buildscript {
```

Comment:
We don't need this, since we're not pulling down any plugins (other than those provided by gradle)

Reply:
done